contentful/export-processor/content-types/section.js

```diff
@@ -152,6 +152,119 @@ export class Section {
+                newPath.unshift({ question: lastQuestion, answer: lastQuestion.answers[0] })
+            }
+
+            // Loop through remaining questions
+            for (let i = this.questions.length - 2; i >= 0; i--) {
+                const currentQuestion = this.questions[i]
+                if (neededNextQuestions.length) {
+                    const validAnswers = currentQuestion.answers.filter(answer => answer.nextQuestion && answer.nextQuestion.sys.id == neededNextQuestions[0])
+                    const validUnusedAnswers = validAnswers.filter(answer => !answersUsed.includes(answer.id))
+
+                    currentAnswer = validUnusedAnswers.length ? validUnusedAnswers[0] : validAnswers[0]
+
+                    if (validUnusedAnswers.length) {
+                        answersUsed.push(currentAnswer.id)
+                    }
+
+                    neededNextQuestions.unshift(currentQuestion.id)
+
+                } else {
+                    const unusedAnswers = currentQuestion.answers.filter(answer => !answersUsed.includes(answer.id))
+
+                    currentAnswer = unusedAnswers.length ? unusedAnswers[0] : currentQuestion.answers[0]
+
+                    if (unusedAnswers.length) {
+                        neededNextQuestions.unshift(currentQuestion.id)
+                        answersUsed.push(currentAnswer.id)
+                    }
+                }
+
+                newPath.unshift({ question: currentQuestion, answer: currentAnswer })
+            }
+
+            // Check through new path and remove later questions if nextQuestion is undefined (ie path contains early answer that shortens the user journey)
+            for (let i = 0; i < newPath.length; i++) {
+                if (newPath[i].answer && !newPath[i].answer.nextQuestion) {
+                    newPath.splice(i + 1, newPath.length - i + 1)
+                    break;
+                } else if (newPath[i].answer && newPath[i].answer.nextQuestion.sys.id !== newPath[i + 1].question.id) {
+                    const nextQuestion = newPath.findIndex((question) => question.question.id === newPath[i].answer.nextQuestion.sys.id, i + 1)
+                    newPath.splice(i + 1, nextQuestion - (i + 1))
+                    continue;
+                }
+            }
+            allAnswerPaths.push(newPath)
+        }
+        this.pathsForAllPossibleAnswers = allAnswerPaths.map((path) => {
+            const userJourney = new UserJourney(path, this);
+            userJourney.setRecommendation(this.recommendation, path);
+            return userJourney;
+        });
+    }
+
+    checkAllChunksTested() {
+        const sectionChunks = this.recommendation.section.chunks;
+        const testedChunks = [];
+
+        Object.values(this.minimumPathsForRecommendations).forEach((path) => {
+            testedChunks.push(this.recommendation.section.getChunksForPath(path));
+        })
+
+        this.pathsForAllPossibleAnswers.forEach((userJourney) => {
```

Comment:
Can you combine this with the above to create, and populate, `testedChunks` in one single allocation?